pkg/kubelet/volume/volume_manager.go

```diff
@@ -102,10 +102,14 @@ type VolumeManager interface {
 	// pod object is bad, and should be avoided.
 	GetVolumesForPodAndAppendSupplementalGroups(pod *api.Pod) container.VolumeMap
 
-	// Returns a list of all volumes that are currently attached according to
-	// the actual state of the world cache and implement the volume.Attacher
-	// interface.
+	// Returns a list of all volumes that implement the volume.Attacher
+	// interface and are currently in use according to the actual and desired
+	// state of the world caches.
```

Comment:
please clarify what "in use" means in general terms, (globally and bind mounted, right?). Not necessarily here, but anywhere prominent.